README.md

```diff
@@ -1,3 +1,32 @@
 # CivicTechExchange
 
-.
+Linux setup:
+
+# install postgress and venv
+sudo apt-get update
+sudo apt-get install postgresql postgresql-contrib
+sudo apt-get install python3-venv
+
+# checkout the repo
+git clone https://github.com/DemocracyLab/CivicTechExchange.git
+
+# create and activate venv
+cd CivicTechExchange/
+python -m venv myenv
+source myvenv/bin/activate
+
+# install python libraries
```

Comment:
Requirements.txt is already a single source of truth for python dependencies, so we should just say:
pip install -r requirements.txt

Reply:
It should have that now.